contracts/src/Verification.sol

```diff
@@ -76,6 +76,9 @@ library Verification {
     uint256 public constant DIGEST_ITEM_PRERUNTIME = 6;
     uint256 public constant DIGEST_ITEM_RUNTIME_ENVIRONMENT_UPDATED = 8;
 
+    /// @dev Prefix of enum variants of DigestItem(0 is reserved for snowbridge)
+    bytes1 public constant DIGEST_ITEM_PREFIX = 0;
```

Comment:
Prefer this naming:
```suggestion
    bytes1 public constant DIGEST_ITEM_OTHER_SNOWBRIDGE = 0x00;
```

Reply:
[d0bb6c9](https://github.com/Snowfork/snowbridge/pull/1053/commits/d0bb6c91d135c6be705ce85f43e166d29b6960a0)